server/cp/ingest/parser/cp_onclusive.py

```diff
@@ -47,39 +51,50 @@ def parse(self, content, provider=None):
         events = []
 
         for item in items:
-            category = []
-            if item.get("subject"):
-                for subject in item["subject"]:
-                    if subject["scheme"] == "onclusive_categories":
-                        onclusive_category = self.find_cv_item(
-                            onclusive_cv_items, subject["qcode"]
-                        )
-                        if onclusive_category:
-                            anpa_category = self.find_cv_item(
-                                anpa_categories, onclusive_category["cp_category"]
+            event = get_resource_service("events").find_one(
+                req=None, guid=item.get("guid")
+            )
+            if event and event.get("version_creator"):
+                # this event is edited by someone so we don't update it
+                continue
+            else:
```

Comment:
This `else:` statement is not required, as the previous line of code is jumping to next iteration.